Azure/sonic-buildimage
# [Inventec][D6332] Add new platform Inventec D6332

**- What I did**
* Add D6332 device files
* Add D6332 driver files
* Add D6332 Platform API implementation
* Modify Makefile, debian rules and add init scripts

**- How I did it**
* Add D6332 device files to device/inventec/x86_64-inventec_d6332-r0/
* Add D6332 driver files to platform/broadcom/sonic-platform-modules-inventec/d6332/modules/
* Add D6332 Platform API implementation to platform/broadcom/sonic-platform-modules-inventec/d6332/sonic_platform
* Modify Makefile, debian rules and add init scripts in platform/broadcom/sonic-platform-modules-inventec/

**- How to verify it**
* check drivers: lsmod |grep -E "^inv_"
* check platform eeprom: show platform syseeprom
* check platform PSUs:  show platform psustatus
* check interface status: show interfaces status
* check transceiver presence: show interfaces transceiver presence

**- Description for the changelog**

* Nov 15, 2019 initial commit
  * Add D6332 device files
  * Add D6332 driver files
  * Add D6332 Platform API implementation
  * Modify Makefile, debian rules and add init scripts

**- A picture of a cute animal (not mandatory but encouraged)**
